UnityProject/Assets/Scripts/Items/Kitchen/Dryable.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using UnityEngine;
+using NaughtyAttributes;
+
+
+public class Dryable : MonoBehaviour
+{
+	[Tooltip("Minimum time to dry.")]
+	public int DryTime = 10;
+
+	[InfoBox("If no transormation is to take place, then don't select any item. Don't select the same item as itself.", EInfoBoxType.Warning)]
+	[Tooltip("What this item becomes when dried." +
+				"If not set, this item will not change GameObject when dried, but will still invoke the dried event.")]
+	public GameObject DriedProduct;
+
+	/// <summary>
+	/// Raised when enough cooking time has been added (via <see cref="AddCookingTime(float)"/>)
```

Comment:
Probably want to adjust `(via <see cref="AddCookingTime(float)"/>)` to be 
`(via <see cref="AddDryingTime(float)"/>)`

Reply:
shit you're right